### PR TITLE
Document admission control plugin configuration

### PR DESCRIPTION
**Pull request by csrwng**
Documents admission control plugins and the configuration exposed in master-config.yaml

### CODE REVIEW

**Comment by csrwng** [comment]
@deads2k @derekwaynecarr ptal
@dgoodwin fyi

File: install_config/configuring_admission_control.adoc [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+[source, yaml]
+----
+- NamespaceLifecycle
+- OriginPodNodeEnvironment
+- LimitRanger
+- ServiceAccount
+- SecurityContextConstraint
+- ResourceQuota
+- SCCExecRestrictions
```
Comment (derekwaynecarr) [inline install_config/configuring_admission_control.adoc]: ResourceQuota should always be last in the sequence.

Please include wording to that affect.

Comment (csrwng) [inline install_config/configuring_admission_control.adoc]: It's not in the code. We should open an issue. 
https://github.com/openshift/origin/blob/master/pkg/cmd/server/kubernetes/master_config.go#L38

File: install_config/configuring_admission_control.adoc [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+link:../dev_guide/quota.html[resource quota] that has been configured for a particular project.
+
+* *SCCExecRestrictions* Prevents users from remotely executing processes on pods that they cannot create according to current
+link:../architecture/additional_concepts/authorization.html#security-context-constraints[security context constraints].
+
+== Providing configuration for Admission Control Plugins
+
+Certain admission control plugins may take an additional configuration file. The configuration for a plugin may be provided either
+as a file location or an embedded configuration object in
+link:master_node_configuration.html#master-configuration-files[master-config.yaml].
```
Comment (sosiouxme) [inline install_config/configuring_admission_control.adoc]: Am I right in thinking that kubernetes plugins can't accept embedded configuration? The interface init method only accepts an io.Reader.

I don't think any actually take config yet, but I'm working on one...

Comment (csrwng) [inline install_config/configuring_admission_control.adoc]: On the startup of our master we write the embedded config to a temp file and pass that file's stream to the plugin.

**Comment by csrwng** [comment]
Closing in favor of #1476

**Pull request closed without merging**